processor/memorylimiterprocessor/config.go

```diff
@@ -1,68 +1,13 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-// Package memorylimiterprocessor provides a processor for OpenTelemetry Service pipeline
-// that refuses data on the pipeline according to the current state of memory usage.
 package memorylimiterprocessor // import "go.opentelemetry.io/collector/processor/memorylimiterprocessor"
 
 import (
-	"errors"
-	"time"
-
 	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/internal/memorylimiter"
 )
 
-var (
-	errCheckIntervalOutOfRange        = errors.New("'check_interval' must be greater than zero")
-	errLimitOutOfRange                = errors.New("'limit_mib' or 'limit_percentage' must be greater than zero")
-	errSpikeLimitOutOfRange           = errors.New("'spike_limit_mib' must be smaller than 'limit_mib'")
-	errSpikeLimitPercentageOutOfRange = errors.New("'spike_limit_percentage' must be smaller than 'limit_percentage'")
-	errLimitPercentageOutOfRange      = errors.New(
-		"'limit_percentage' and 'spike_limit_percentage' must be greater than zero and less than or equal to hundred")
-)
-
-// Config defines configuration for memory memoryLimiter processor.
-type Config struct {
-	// CheckInterval is the time between measurements of memory usage for the
-	// purposes of avoiding going over the limits. Defaults to zero, so no
-	// checks will be performed.
-	CheckInterval time.Duration `mapstructure:"check_interval"`
-
-	// MemoryLimitMiB is the maximum amount of memory, in MiB, targeted to be
-	// allocated by the process.
-	MemoryLimitMiB uint32 `mapstructure:"limit_mib"`
-
-	// MemorySpikeLimitMiB is the maximum, in MiB, spike expected between the
-	// measurements of memory usage.
-	MemorySpikeLimitMiB uint32 `mapstructure:"spike_limit_mib"`
-
-	// MemoryLimitPercentage is the maximum amount of memory, in %, targeted to be
-	// allocated by the process. The fixed memory settings MemoryLimitMiB has a higher precedence.
-	MemoryLimitPercentage uint32 `mapstructure:"limit_percentage"`
-
-	// MemorySpikePercentage is the maximum, in percents against the total memory,
-	// spike expected between the measurements of memory usage.
-	MemorySpikePercentage uint32 `mapstructure:"spike_limit_percentage"`
-}
+type Config memorylimiter.Config
```

Comment:
This is a breaking change. It has to an alias
```suggestion
var Config = memorylimiter.Config
```

Reply:
i may missing something about golang. but how can you make a alias with var?